projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -268,11 +265,26 @@ export function ExtrasInterstitial() {
  * @returns {object} JetpackAIInterstitial react component.
  */
 export function JetpackAIInterstitial() {
+	const slug = 'jetpack-ai';
+	const { detail } = useProduct( slug );
+	const { onClickGoBack } = useGoBack( { slug } );
+
+	const currentTier = detail?.[ 'ai-assistant-feature' ]?.[ 'current-tier' ]?.value;
+	const hasNextTier = ! [ 1, 500 ].includes( currentTier );
+
+	if ( ! hasNextTier ) {
+		return <JetpackAIInterstitialMoreRequests onClickGoBack={ onClickGoBack } />;
```

Comment:
In this case, are we going to show the `JetpackAIInterstitialMoreRequests` when the user has the unlimited plan?

Reply:
As it is, yes, but the page is not linked to once a user has a plan, so we need to think about how to migrate users.
Which reminds me that the `Manage` button should be upgraded as well.